holoviews/streams.py

```diff
@@ -223,3 +223,60 @@ class PositionXY(Stream):
     def __init__(self, preprocessors=[], source=None, subscribers=[], **params):
         super(PositionXY, self).__init__(preprocessors=preprocessors, source=source,
                                         subscribers=subscribers, **params)
+
+
+
+
+class ParamValues(Stream):
+    """
+    A Stream based on the parameter values of some other parameterized
+    object, whether it is a parameterized class or a parameterized
+    instance.
+
+    The update method enables the stream to update the parameters of the
+    specified object.
+    """
```

Comment:
Can the docstring be improved?  The PR makes it sound like the stream is updating the parameters, primarily, but what I thought was that it went the other way around, and here it seems to do both.  Are these two very different usages?  Seems like something to explain.